examples/src/simple-pointcloud.ts

```diff
@@ -3,31 +3,30 @@
  */
 
 import * as THREE from 'three';
-// @ts-ignore
-import * as Potree from '@cognite/potree-core';
 import * as reveal from '@cognite/reveal';
 import * as reveal_threejs from '@cognite/reveal/threejs';
 
 import CameraControls from 'camera-controls';
 import dat from 'dat.gui';
 import { vec3 } from 'gl-matrix';
+import { CogniteClient } from '@cognite/sdk';
+import { toThreeJsBox3 } from '@cognite/reveal/threejs';
 
 CameraControls.install({ THREE });
 
 async function main() {
-  const modelUrl = new URL(location.href).searchParams.get('model') || '/transformer-point-cloud/cloud.js';
+  const urlParams = new URL(location.href).searchParams;
+  const modelIdentifier = urlParams.get('model') || '/transformer-point-cloud/cloud.js';
+  const project = urlParams.get('project') || '3ddemo';
```

Comment:
Do not leak internal tenant names. Only we have access to this tenant.

Reply:
Good point :)